platform/darwin/src/MGLTypes.h

```diff
@@ -30,6 +30,11 @@ NS_ASSUME_NONNULL_BEGIN
     typedef NSString *NSNotificationName;
 #endif
 
+typedef NSString *MGLExceptionName NS_EXTENSIBLE_STRING_ENUM;
```

Comment:
I was copying `NSExceptionName`, but I’m going to look at replacing this with `NS_TYPED_EXTENSIBLE_ENUM` on [Apple’s recommendation](https://developer.apple.com/documentation/swift/objective_c_and_c_code_customization/grouping_related_objective_c_constants):

> You might encounter Objective-C code that uses the older `NS_STRING_ENUM` and `NS_EXTENSIBLE_STRING_ENUM` macros, which were used to group string constants. Use `NS_TYPED_ENUM` and `NS_TYPED_EXTENSIBLE_ENUM` when grouping related constants of any type, including string constants.

Reply:
Xcode 9 has all of these defined as the same thing, so it’s just cosmetic:

```
#define NS_STRING_ENUM _NS_TYPED_ENUM
#define NS_EXTENSIBLE_STRING_ENUM _NS_TYPED_EXTENSIBLE_ENUM

#define NS_TYPED_ENUM _NS_TYPED_ENUM
#define NS_TYPED_EXTENSIBLE_ENUM _NS_TYPED_EXTENSIBLE_ENUM
```

... but let’s still standardize on the new one.